packages/Field/styles.ts

```diff
@@ -17,7 +19,14 @@ const checkableFieldStyles = css`
   margin-bottom: sm;
 `
 
-export const Field = styled('div').withConfig({ shouldForwardProp })(
+type StyledFieldProps = {
+  checkableField: boolean
+  flexDirection: flexDirection
+  checked: boolean
+  size: Size
```

Comment:
All is required ?

Reply:
Oui, toutes les props lui sont passées sauf erreur de ma part